### PR TITLE
Add unittest for describe secret

### PR DESCRIPTION
**Pull request by zhangxiaoyu-zidif**
**What this PR does / why we need it**:
add unittest for describe secret

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by zhangxiaoyu-zidif** [comment]
@fabianofranz 
PTAL

File: pkg/printers/internalversion/describe_test.go [hunk 1]
```diff
@@ -118,6 +118,28 @@ func TestDescribePodTolerations(t *testing.T) {
...
+			"password": []byte("MWYyZDFlMmU2N2Rm"),
+		},
+	})
+	c := &describeClient{T: t, Namespace: "foo", Interface: fake}
+	d := SecretDescriber{c}
+	out, err := d.Describe("foo", "bar", printers.DescriberSettings{})
+	if err != nil {
+		t.Errorf("unexpected error: %v", err)
+	}
+	if !strings.Contains(out, "bar") || !strings.Contains(out, "foo") || !strings.Contains(out, "username") || !strings.Contains(out, "8 bytes") || !strings.Contains(out, "password") || !strings.Contains(out, "16 bytes") {
```
Comment (fabianofranz) [inline pkg/printers/internalversion/describe_test.go]: You might want to make it explicitly sure the describer is not including the sensitive data, e.g `|| strings.Contains(out, MWYyZDFlMmU2N2Rm)`.

Comment (zhangxiaoyu-zidif) [inline pkg/printers/internalversion/describe_test.go]: OK，got it. I would fix soon.

Comment (zhangxiaoyu-zidif) [inline pkg/printers/internalversion/describe_test.go]: Yes, sir. I just check the that only shows number of char. of classified info.

Comment (zhangxiaoyu-zidif) [inline pkg/printers/internalversion/describe_test.go]: And need I add some test to check if these sensitive info exist?

Comment (fabianofranz) [inline pkg/printers/internalversion/describe_test.go]: Yes, explicitly check for the presence of the sensitive data that's not supposed to be present on the output, and error out if it's there.

**Comment by zhangxiaoyu-zidif** [comment]
@fabianofranz
Fixed. PTAL

**Comment by zhangxiaoyu-zidif** [comment]
@fabianofranz 
PTAL

**Comment by zhangxiaoyu-zidif** [comment]
ping @fabianofranz

**Comment by zhangxiaoyu-zidif** [comment]
@fabianofranz 
PTAL. Thanks!

**Comment by fabianofranz** [comment]
Thanks @zhangxiaoyu-zidif!
/lgtm
/approve no-issue
/sig cli

**Pull request merged**